src/data/quizzes/questionBank.ts

```diff
@@ -887,6 +887,6 @@ const questionBank: QuestionBank = {
     ],
     correctAnswerId: "h005-b",
   },
-}
+} as const satisfies QuestionBank
```

Comment:
Is there a benefit doing `as const satisfies` instead of just `satisfies`?

Reply:
I tried without, but had to add back `as const` ([in this commit](https://github.com/ethereum/ethereum-org-website/pull/11992/commits/e5d5f1729624de9a4301910b66745a7af3d94715)) after it wasn't casting the `typeof` as literals to perform `keyof` on... instead it was just simplified to `string` (IIRC)...

The other place `satisfies` is used didn't give me that problem, so I kept `as const` off of that one.